### PR TITLE
[tw] Add content/zh-tw/agile-software-development.md

### LINKED ISSUE #2153: [tw] Localize `Agile Software Development` for Traditional Chinese
+ Localize `Agile Software Development` in Traditional Chinese
+ https://glossary.cncf.io/agile-software-development/

### PR DESCRIPTION
**Pull request by gamerslouis**
### Describe your changes
Origin content from https://glossary.cncf.io/agile-software-development/

### Related issue number or link (ex: `resolves #issue-number`)
resolve #2153 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by hwchiu — COMMENTED** [review]
感覺比較常看見 迭代 而非疊代 (Iterative)

話說有沒有原文連結？

**Review by pichuang — APPROVED** [review]
lgtm

File: content/zh-tw/agile-software-development.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: 敏捷軟體開發
+status: Completed
+category: 概念
+tags: ["方法論", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+敏捷軟體開發是一系列強調迭代開發週期和自我組織團隊的實踐方法。與僅在專案結束時才生成價值的瀑布式專案相比，敏捷軟體開發著重於價值的持續、增量式的交付，並對過程本身進行漸進式的改善。
```
Comment (ydFu) [inline content/zh-tw/agile-software-development.md:10]:
```suggestion
敏捷軟體開發是一系列強調迭代開發週期和自我組織團隊的實踐方法。
與僅在專案結束時才生成價值的瀑布式專案相比，
敏捷軟體開發著重於價值的持續、增量式的交付，並對過程本身進行漸進式的改善。
```

File: content/zh-tw/agile-software-development.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+在軟體專案中，定義、傳達和理解所有利害關係人的需求非常困難，甚至是不可能的。然而，客戶希望他們的軟體專案能有高品質，在預算、時程及專案範圍內交付。相較於於瀑布式策略，敏捷軟體開發的週期特性使其能持續適應需求，並且能更快地適應其他所有情況。
```
Comment (ydFu) [inline content/zh-tw/agile-software-development.md:14]:
```suggestion
在軟體專案中，定義、傳達和理解所有利害關係人的需求非常困難，甚至是不可能的。
然而，客戶希望他們的軟體專案能有高品質，在預算、時程及專案範圍內交付。
相較於於瀑布式策略，敏捷軟體開發的週期特性使其能持續適應需求，並且能更快地適應其他所有情況。
```

File: content/zh-tw/agile-software-development.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## 如何幫助 {#how-it-helps}
+
+敏捷軟體開發包含了傳統（瀑布式）策略的所有階段，如需求工程、規劃、開發、審核、測試和交付。最大的不同在於，整個軟體專案的時間跨度被切分為多個迭代，每個迭代都包含所有這些階段。在每個迭代結束後，開發團隊可以與客戶一起審查創造的價值，並對需求進行調整以達到最終目標。此外，開發團隊還會反思需要採取哪些行動來改進過程本身。
```
Comment (ydFu) [inline content/zh-tw/agile-software-development.md:18]:
```suggestion
敏捷軟體開發包含了傳統（瀑布式）策略的所有階段，
如需求工程、規劃、開發、審核、測試和交付。
最大的不同在於，整個軟體專案的時間跨度被切分為多個迭代，
每個迭代都包含所有這些階段。
在每個迭代結束後，開發團隊可以與客戶一起審查創造的價值，並對需求進行調整以達到最終目標。
此外，開發團隊還會反思需要採取哪些行動來改進過程本身。
```

**Review by ydFu — COMMENTED** [review]
In accordance with the [style guide](https://glossary.cncf.io/style-guide/#three-subheadings) set forth by the cncf/glossary,please use semantic line breaks (one sentence per line)

**Comment by ydFu** [comment]
This seems to be moved to other unrelated files

File: content/zh-tw/agile-software-development.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+在軟體專案中，定義、傳達和理解所有利害關係人的需求非常困難，甚至是不可能的。
+然而，客戶希望他們的軟體專案能有高品質，在預算、時程及專案範圍內交付。
+相較於於瀑布式策略，敏捷軟體開發的週期特性使其能持續適應需求，並且能更快地適應其他所有情況。
+
+## 如何幫助 {#how-it-helps}
+
+敏捷軟體開發包含了傳統（瀑布式）策略的所有階段，如需求工程、規劃、開發、審核、測試和交付。
+最大的不同在於，整個軟體專案的時間跨度被切分為多個迭代，每個迭代都包含所有這些階段。
+在每個迭代結束後，開發團隊可以與客戶一起審查創造的價值，並對需求進行調整以達到最終目標。
+此外，開發團隊還會反思需要採取哪些行動來改進過程本身。
```
Comment (ydFu) [inline content/zh-tw/agile-software-development.md:25]:
```suggestion
敏捷軟體開發包含了傳統（瀑布式）策略的所有階段，
如需求工程、規劃、開發、審核、測試和交付。
最大的不同在於，整個軟體專案的時間跨度被切分為多個迭代，
每個迭代都包含所有這些階段。
在每個迭代結束後，開發團隊可以與客戶一起審查創造的價值，並對需求進行調整以達到最終目標。
此外，開發團隊還會反思需要採取哪些行動來改進過程本身。
```

File: content/zh-tw/agile-software-development.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+與僅在專案結束時才生成價值的瀑布式專案相比，
+敏捷軟體開發著重於價值的持續、增量式的交付，並對過程本身進行漸進式的改善。
+
+## 解決的問題 {#problem-it-addresses}
+
+在軟體專案中，定義、傳達和理解所有利害關係人的需求非常困難，甚至是不可能的。
+然而，客戶希望他們的軟體專案能有高品質，在預算、時程及專案範圍內交付。
+相較於於瀑布式策略，敏捷軟體開發的週期特性使其能持續適應需求，並且能更快地適應其他所有情況。
+
+## 如何幫助 {#how-it-helps}
```
Comment (ydFu) [inline content/zh-tw/agile-software-development.md:20]:
```suggestion
## 如何幫助我們 {#how-it-helps}
```
Modified based on discussion in [slack](https://cloud-native.slack.com/archives/C055KERTNCX/p1692440415536599?thread_ts=1692324528.708459&cid=C055KERTNCX)

**Review by ydFu — COMMENTED** [review]
Use rebase to squash/fixup dummy/unnecessary commits into only one commit.

**Review by ydFu — APPROVED** [review]
LGTM, thanks @gamerslouis

**Pull request merged**